frontend/@mf-types/@mf/modelRegistry/compiled-types/src/app/pages/settings/ModelRegistryTableRowStatus.d.ts

```
@@ -0,0 +1,7 @@
+import React from 'react';
+import { K8sCondition } from 'mod-arch-shared';
+interface ModelRegistryTableRowStatusProps {
+    conditions: K8sCondition[] | undefined;
+}
+export declare const ModelRegistryTableRowStatus: React.FC<ModelRegistryTableRowStatusProps>;
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)